libs/utils/utils.js

```diff
@@ -104,6 +105,17 @@ export async function loadBlock(block) {
   return block;
 }
 
+export async function loadLCP({ blocks, lcpList, loader }) {
+  const list = lcpList || LCP_BLOCKS;
+  const blockLoader = loader || loadBlock;
```

Comment:
minor: use default params
```suggestion
export async function loadLCP({ blocks = [], lcpList = LCP_BLOCKS, loader = loadBlock }) {
```

Reply:
Fixed.